src/quest/persistence.py

```diff
@@ -52,6 +52,13 @@ def remove(self, item: EventRecord):
         self._storage.delete_blob(key)
         self._storage.write_blob(self._namespace, self._keys)
 
+    def clear(self):
+        for key in self._keys:
+            self._storage.delete_blob(key)
+        self._items.clear()
+        self._keys.clear()
+        self._storage.delete_blob(self._namespace)
+
```

Comment:
Anything more special that this? On the passing test this wiped out the records. All that was left was the base directory.